src/python/pants/backend/python/goals/export_test.py

```diff
@@ -233,7 +257,7 @@ def test_export_venv_new_codepath(
             assert ppc0.extra_env["PEX_MODULE"] == "pex.tools"
             assert ppc0.extra_env.get("PEX_ROOT") is not None
 
-            ppc1 = result.post_processing_cmds[1]
+            ppc1 = result.post_processing_cmds[-1]
```

Comment:
One of my previous commits changed this from `ppc1` to `ppc_last` - would that be beneficial in avoiding confusion here?

Reply:
ah no, the confusion was in my (wrong) understanding at the beginning of the comment that the `rm -rf` post processing command would always be the second element in the list. With that misunderstanding cleared up this change makes sense. Not sure why I left the comment rather than remove it--maybe to show my train of review finding out about stuff.